apps/errant-tower-client/src/common/styles/variables.scss

```
@@ -0,0 +1,92 @@
+// Colors
+$color-primary-100: #fef3c6;
+$color-primary-200: #fde9ab;
+$color-primary-300: #ffd869;
+$color-primary-400: #ffb900;
+$color-primary-500: #fe9a00;
+$color-primary-600: #e17100;
+$color-primary-700: #bb4d00;
+$color-primary-800: #963c00;
+$color-primary-900: #7b3306;
+
+$color-secondary-100: #dff2fe;
```

Stylelint (17.7.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)